ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json

```diff
@@ -489,5 +489,88 @@
+      "paragraphText5part2": "The types of questions we strive to answer include:",
+      "paragraphText5list1": "What are all the genes and mutations that can lead to leiomyosarcoma?",
+      "paragraphText5list2": "What explains why some patients show extraordinary responses to a particular\ntreatment?",
+      "paragraphText5list3": "What explains why some tumors never respond to a particular treatment?",
+      "paragraphText5list4": "How can we develop better treatments for leiomyosarcoma?",
+      "paragraphText5part3": "Despite the progress that has been made to begin to answer these questions, we remain far\nfrom the goal. To get there, the detailed genomic characterization of many thousands of\nclinically annotated cancer samples will be required.",
+      "paragraphTitle6": "What is the goal of the Leiomyosarcoma project?",
+      "paragraphText6": "The goal of the LMSproject is to transform cancer care by enabling all patients who have ever been diagnosed with leiomyosarcoma to accelerate biomedical research through sharing their cancer samples, clinical information, and their experiences. In order to achieve these goals, datasets containing linked clinical, genomic, molecular, and patient-reported data (without including any personal information) will be regularly shared with the biomedical research community through scientific repositories such as cBioPortal for Cancer Genomics and the National Cancer Institute (NCI) Genomic Data Commons\n",
+      "paragraphTitle7": "What steps are involved for participants?",
+      "paragraphText7": "Participants register for the study online at LMSproject.org. By consenting to the study, patients allow us to request and obtain copies of their medical records, archived tumor samples (if available), and a saliva and optional blood sample. If desired, the participant may also opt-in to providing an additional saliva sample to learn more about their germline (or “normal”) DNA, described in further detail later on this page. ",
+      "paragraphTitle8": "What steps are involved for the physician’s offices of enrolled participants?\n",
+      "paragraphText8": "When they enroll, participants provide information about where they have received care for their leiomyosarcoma. The study team works directly with medical record and pathology departments to obtain copies of medical records and portions of archived tumor samples. We will ask pathology departments to share only a part of tumor tissue, and not to share anything with us that might be needed for clinical care. \n",
+      "paragraphText5part4": "If you have any questions about this study, please reach out to us at info@lmsproject.org or XXX-XXX-XXXX.\n",
```

Comment:
Make this `info@lmsproject.org` email text as an email link.

There should be a real number instead of `XXX-XXX-XXXX`

Reply:
done